docassemble/AssemblyLine/al_document.py

```diff
@@ -396,7 +396,7 @@ class ALDocument(DADict):
     - filename: name used for output PDF
     - title: display name for the output PDF
     - enabled
-    - has_addendum: set to False if the document never has overflow, like for a DOCX template
+    - has_addendum: The default value is False. Set to True if the document has overflow, like for a DOCX template.
```

Comment:
The docstring now says the complete opposite as it did before. Before, it said that DOCX files shouldn't have an addendum, but now it's saying that DOCX files should have an addendum?

Reply:
Good catch! Yes, DOCX files would be weird to have an addendum

Reply:
Thanks @BryceStevenWilley !